bokeh/models/widgets/markups.py

```diff
@@ -103,6 +106,11 @@ class Div(Markup):
     The default value is ``False``, meaning contents are rendered as HTML.
     """)
 
+    enable_tex = Bool(False, help="""
+    Whether the contents should be rendered including TeX/LaTeX input.
+    The default value is ``False``, and only applies when rendering as HTML (that is, when render_as_text is ``False``)
```

Comment:
line length, also no need to code-ify True and False
```suggestion
    The default value is False, and only applies when rendering as HTML 
    (that is, when ``render_as_text`` is False)
```